app/models/container.js

```diff
@@ -38,6 +39,37 @@ var Container = Resource.extend(DisplayImage, {
     return choices;
   }),
 
+  validateQuota() {
+    const projectLimit =  get(this, 'scope.currentProject.resourceQuota.limit');
+
+    if ( !projectLimit ) {
+      return [];
+    }
+
+    const intl = get(this, 'intl');
+    const errors = [];
+
+    const limitsCpu = get(projectLimit, 'limitsCpu');
```

Comment:
You can utilize destructuring assignment here. Its not required but I figured you may not know about it yet and it makes the code a little more concise imo. 

```
const { limitsCpu, limitsMemory, requestCpu, requestMemory } = projectLimit;
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment